util/gnmi.go

```diff
@@ -43,6 +43,8 @@ func PathMatchesPrefix(path *gpb.Path, prefix []string) bool {
 }
 
 // PathElemsEqual replaces the proto.Equal() check for PathElems.
+// Note: Use PathElemsEquivalent for wildcards.
```

Comment:
PathElemsEquivalent -> PathMatchesQuery

Reply:
moved  that comment to the PathMatchesPathElemPrefix